client/src/components/CardDialog/cardDialogStyles.ts

```diff
@@ -93,6 +69,30 @@ const cardDialogStyles = makeStyles((theme: Theme) => ({
     height: 8,
     borderRadius: 9999,
   },
+  columnButton: {
+    backgroundColor: theme.palette.secondary.light,
+    color: theme.palette.secondary.main,
+    padding: '5px 0',
+    margin: '2px 5px',
+    border: 'none',
+    width: '90px',
+    '&:hover': {
+      backgroundColor: theme.palette.primary.main,
+      color: theme.palette.primary.contrastText,
+    },
+  },
+  columnButtonActive: {
+    backgroundColor: theme.palette.primary.main,
+    color: theme.palette.primary.contrastText,
+    padding: '5px 0',
+    margin: '2px 5px',
+    border: 'none',
+    width: '90px',
+    '&:hover': {
+      backgroundColor: theme.palette.secondary.light,
+      color: theme.palette.secondary.main,
+    },
+  },
```

Comment:
Are these styles the same as the above file @street-jackal? If so, you could merge them into a single `useStyles` and grab the same `useStyles` for different files to use!

Reply:
Good catch. There are a lot of similarly repeated styles in my code. I've been meaning to to a batch update / combine of these but haven't gotten around to it. I'll give it a first pass this morning!